server.js

```diff
@@ -1,19 +1,23 @@
 'use strict';
 
 const path = require('path');
+const bodyParser = require('body-parser');
 const app = require('express')();
 const tasksContainer = require('./tasks.json');
 const webpack = require('webpack');
 const config = require('./webpack.config.dev');
 const compiler = webpack(config);
 let lastTaskId = tasksContainer.tasks.length + 1;
 
+const findTask = id => tasksContainer.tasks.find(item => item.id === id) || null;
```

Comment:
the `find` method returns `null` if no items match your condition. Why did you add `|| null` at the end of your function?